### DISCUSSION TITLE
Curation report April 10 - April 14

### DISCUSSION THREAD

**Discussion by ashviyer** [wikipathways/wikipathways-help#30] [curation-reports]
Hi everyone, my first time as a curator of the week.

1. Underconstruction and needs work pathways:
a.  [WP5094](https://classic.wikipathways.org/index.php/Pathway:WP5094): - Lots of changes in this pathway with correcting annotations and database selection for datanodes. 
Note: - The miss-annotation of nodes does not get detected on the WikiPathways database, I guess. Because I only came across it manually going through all the nodes on PathVisio. 

- Also, the removal of the colored datanodes based on whether they are under- or over-expressed did not seem necessary for the pathway on the database. Started a discussion with the author of the pathway regarding this.

b.  [WP5353](https://classic.wikipathways.org/index.php/Pathway:WP5353) - This pathway had similar issues with annotations and database selection for nodes. Issues with also datanodes being used to describe text, which was corrected.
- The datanodes do not conform with the standard practices. Have started a conversation with the author.
- Added ontology for this pathway.

Additionally, on the WikiPathways site, some of the nodes for both these pathways dont have a pop-up that displays the identifiers although the datanodes are appropriately annotated. Not sure what the issue is.

3. Reactome pathways: For the Reactome pathways in general, I saw that the pathway nodes are not the standard pathway node on the PathVisio. Is this kept deliberately different? [WP3569](https://classic.wikipathways.org/index.php/Pathway:WP3569)

4. Other pathways required minor corrections regarding adding onology terms and converting the datanodes to graphical elements wherever necessary. 

Thank you.

**Comment by khanspers** [comment]
Thank you @aish181095 !  A few comments/answers:

- [WP5094](https://classic.wikipathways.org/index.php/Pathway:WP5094) and [WP5353](https://classic.wikipathways.org/index.php/Pathway:WP5353): These two are both from the same group, and are in the style of [NetPath pathways](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-1-r3). WikiPathways has many NetPath pathways and they frequently have similar issues (color-filled data nodes, unconnected interactions etc), so many of them are not approved. The PI (Dr. Prasad) is aware of these issues and has recently stated that their curators will be making the necessary updates.

- For Reactome pathways, they were constructed using a converter tool (described [here](https://doi.org/10.1371/journal.pcbi.1004941)), and have a particular style. For example the nodes representing complexes (brown hexagons) are specific to the Reactome pathways, but the pathway nodes (green text) are standard as far as I know. Sometimes authors like to add a border to pathway nodes, so you sometimes see them as a green rectangle with green text in other pathways.

- Regarding the missing node annotation pop-ups, can you give an example? 

Also, we are moving the weekly curator's reports to [wikipathways-help](https://github.com/wikipathways/wikipathways-help/discussions) (this is linked to from the QA protocol), so I will move this discussion over to that repo.

Kristina

**Comment by ashviyer** [comment]
Hi @khanspers , thanks for the clarifications.
There is one NetPath pathway that is approved but has similar issues with missing cross-reference : [WP5143](https://classic.wikipathways.org/index.php/Pathway:WP5143). The tag for the pathway should probably be changed?

Regarding the pop-ups for the [WP5094](https://classic.wikipathways.org/index.php/Pathway:WP5094) pathways, when clicking on PRKCA/ PRKCD/PRKCE/PRKCB does not show the identifiers for the node.


![image](https://user-images.githubusercontent.com/37527519/232123866-48268fcb-3771-4545-b0b2-501ad4efca0d.png)

**Reply by khanspers** [reply]
Thank you @aish181095. After some testing there are many more nodes in that pathway that are not clickable, and its most likely due to a known bug with elbow interactions blocking nodes: https://github.com/wikipathways/wikipathways-development/issues/114. I will add a comment to that issue.

**Comment by AlexanderPico** [comment]
Great report! By the way @aish181095, you have a profile page on the new site: https://www.wikipathways.org/authors/Ash%20iyer.html

Feel free to edit it to include more details and links.

**Reply by ashviyer** [reply]
Thank you, @AlexanderPico !

**Comment by khanspers** [comment]
For the issue with non-clickable nodes, there is a quick fix if you come across other pathways with the same problem. See steps below. 

1. Open pathways in PV
2. Select all data nodes (Right-click, Select > Select all DataNodes)
3. Right-click on a node (with all nodes selected) and choose Order > Bring to front.

A feature request for PathVision has been suggested as a more permanent solution: https://github.com/PathVisio/pathvisio/issues/192

**Reply by ashviyer** [reply]
Thanks for the solution!
